spec/views/login_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+RSpec.feature 'Logins', type: :feature do
+  background { visit new_user_session_path }
+
+  it 'displays email field' do
+    expect(page).to have_field('user[email]')
+  end
+
+  it 'display Password field' do
+    expect(page).to have_field('user[password]')
+  end
+
+  it 'Display Login' do
+    expect(page).to have_content 'Log in'
+  end
+
+  context 'Submit a form' do
+    it 'Submit form without email and password' do
```

Comment:
- [ ] It will be nice to re-check the content of this statement. I think it should be `should not submit form without email and password`